src/renderers/shaders/ShaderLib/meshphysical.glsl.js

```diff
@@ -75,16 +74,13 @@ uniform float opacity;
 
 #ifdef IOR
 	uniform float ior;
-#endif
-
-#ifdef USE_SPECULAR
```

Comment:
So the `IOR` define now enables specular uniforms? I don't think this change is acceptable, imo.

Reply:
Yes, I agree with you, but do you reckon one of these macros is redundant? IOR and USE_SPECULAR are enabled simultaneously.

Reply:
Or we can use PHYSICAL instead

Reply:
> IOR and USE_SPECULAR are enabled simultaneously.

Yes. I think was for for readability reasons and I don't think having multiple defines for enabling semantically different uniforms is an issue.

@WestLangley How do you think about this?

Reply:
I think we wanted the `ifdef`'s to test material _properties_, not test for the material itself, so I would not be in favor of `ifdef( PHYSICAL )`.

I would continue to handle the properties separately. I think the code is more clear that way.

Reply:
> I think we wanted the `ifdef`'s to test material _properties_, not test for the material itself, so I would not be in favor of `ifdef( PHYSICAL )`.
> 
> I would continue to handle the properties separately. I think the code is more clear that way.

Understood, Thanks